src/qcodes/dataset/dond/do_2d.py

```diff
@@ -57,21 +34,7 @@ def do2d(
     num_points2: int,
     delay2: float,
     *param_meas: ParamMeasT,
-    set_before_sweep: bool | None = True,
```

Comment:
flush_columns and set_before_sweep are not supported by dond yet. Need to figure out how to deal with that